From fc60f8ae916baf49729e9ce64562407173ab3433 Mon Sep 17 00:00:00 2001
From: Tim Schumacher <timschumi2@arcor.de>
Date: Mon, 1 May 2017 18:51:11 +0200
Subject: [PATCH 20/74] mirror: Updated to 01-May-17 16:52 UTC

 - Added android_device_xiaomi_capricorn
 - Added contributors-cloud-generator
 - Added scripts

Change-Id: I06e5f2493639fa7c16f1a9d4bb2036048aa27ad8
---
 default.xml | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/default.xml b/default.xml
index 2211606..a3b6f30 100644
--- a/default.xml
+++ b/default.xml
@@ -738,6 +738,7 @@
   <project name="LineageOS/android_device_xiaomi_aries" />
   <project name="LineageOS/android_device_xiaomi_armani" />
   <project name="LineageOS/android_device_xiaomi_cancro" />
+  <project name="LineageOS/android_device_xiaomi_capricorn" />
   <project name="LineageOS/android_device_xiaomi_ferrari" />
   <project name="LineageOS/android_device_xiaomi_gemini" />
   <project name="LineageOS/android_device_xiaomi_hydrogen" />
@@ -1681,6 +1682,7 @@
   <project name="LineageOS/cm-less" />
   <project name="LineageOS/cm_platform_sdk" />
   <project name="LineageOS/CMStatsServer" />
+  <project name="LineageOS/contributors-cloud-generator" />
   <project name="LineageOS/ctso_supplicant" />
   <project name="LineageOS/cve_tracker" />
   <project name="LineageOS/device_htc_marvel" />
@@ -1727,6 +1729,7 @@
   <project name="LineageOS/samsung-kernel-c1" />
   <project name="LineageOS/samsung-kernel-galaxys" />
   <project name="LineageOS/samsung-kernel-hercules" />
+  <project name="LineageOS/scripts" />
   <project name="LineageOS/semc-kernel-msm7x30" />
   <project name="LineageOS/snapdragon_swe_engine" />
   <project name="LineageOS/sony-kernel-msm8660" />
-- 
2.17.1

